packages/components/src/unit-control/types.ts

```diff
@@ -76,7 +76,7 @@ export type UnitSelectControlProps = {
 export type UnitControlProps = Omit< UnitSelectControlProps, 'unit' > &
 	Pick<
 		InputControlProps,
-		'hideLabelFromVision' | '__next36pxDefaultSize'
+		'hideLabelFromVision' | 'prefix' | '__next36pxDefaultSize'
```

Comment:
Very cool, I didn't realise that `InputControl` had a `prefix` prop we can expose in `UnitControl` 👍

Reply:
I can see why it might not have been exposed initially as it's a bit of an edge case using a prefix with a UnitControl.  That said, it's a perfect fit for the `BorderControl`.